src/assertions/assertions.csproj

```diff
@@ -7,11 +7,12 @@
         <RootNamespace>Neo.Assertions</RootNamespace>
         <EmbedUntrackedSources>true</EmbedUntrackedSources>
         <DebugType>portable</DebugType>
+        <TargetFramework>net8.0</TargetFramework>
```

Comment:
You can remove all these from the `*.csproj` files the `Directory.Build.props` handles it.

Reply:
fixed